.github/workflows/reusable_build_and_upload_rerun_cli.yml

```diff
@@ -59,7 +59,7 @@ permissions:
 jobs:
   set-config:
     name: Set Config (${{ inputs.PLATFORM }})
-    runs-on: ubuntu-latest-16-cores
+    runs-on: ubuntu-latest
```

Comment:
Was it intentional to nerf this runner?

Reply:
yeah it just sets config, doesn't make sense for it to have more cores I think?